osrf_testing_tools_cpp/src/memory_tools/CMakeLists.txt

```diff
@@ -48,9 +48,20 @@ if(BFD_LIBRARY)
   target_compile_definitions(memory_tools_interpose PRIVATE BACKWARD_HAS_BFD=1)
 endif()
 
+option(ENABLE_MEMORY_TOOLS_PRELOAD
```

Comment:
It's a bit bikeshed, but I think `DISABLE_MEMORY_TOOLS` defaulting to `OFF` would make more sense. I think the "action" is to disable it, since it is enabled by default, whereas taking an action to "unenable" it by passing an option seems strange to me. Also, the mechanism is removing the ld preload, but effectively it completely disables memory tools, basically nothing will work without it, so dropping the `PRELOAD` part made sense to me.

It's ridiculous, but I probably would also have done `OSRF_TESTING_TOOLS_CPP_` before it, so it doesn't have a chance of colliding with another package.

Definitely not a blocker, just something I wanted to bring up in case you agree.

Reply:
Both suggestions make sense to me. I made my case for the ENABLE vs DISABLE option [here](https://github.com/osrf/osrf_testing_tools_cpp/pull/28#issuecomment-473388700) but I'm more swayed by your argument than my own.

I'll make the necessary changes and as long as CI doesn't complain will merge and integrate the changes downstream.

Reply:
Sorry I missed that mention in your older comment, you're free to use which ever you like.